wagtailvideos/jinja2tags.py

```diff
@@ -1,22 +1,14 @@
 from __future__ import absolute_import, unicode_literals
 
-import mimetypes
-
 from jinja2.ext import Extension
 
-from django import template
-from django.forms.widgets import flatatt
-from django.utils.text import mark_safe
-
+from .models import Video
 
-def video(video, attrs):
-    if not video:
-        return ''
 
-    if attrs:
-        return video.video_tag(attrs)
-    else:
-        return video
+def video(video, attrs='preload controls'):
```

Comment:
`attrs` should be a dict, so you can do something like:
```jinja2
{{ video(foo.video, bar='baz', quux=foo.some_attr) }}
```
without having to do strange string building. You can have default values for `preload` and `controls` by doing something like:
```python
def video(video, **attrs):
    defaults = {'preload': True, 'controls': True}
    defaults.update(attrs)
    video.video_tag(attrs)
```
This matches how [Wagtail does its `image` tag in Jinja2](https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailimages/jinja2tags.py#L8) as well: